interfaces/inc/c_pal/file_util.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#ifndef FILE_UTIL_H
+#define FILE_UTIL_H
+
+#ifdef __cplusplus
+#include <cstdint>
+#else
+#include <stdbool.h>
+#include <stdint.h>
+#endif
+
+#include "umock_c/umock_c_prod.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef WIN32 
+#include "windows.h"
+#else
+#include "c_pal/windows_defines.h"
+#endif
+
+MOCKABLE_FUNCTION(, HANDLE, file_util_create_file, const char*, lp_file_name, uint32_t, desired_access, uint32_t, share_mode, LPSECURITY_ATTRIBUTES, lp_security_attributes, uint32_t, creation_disposition, uint32_t, flags_and_attributes, HANDLE, h_template_file);
```

Comment:
>lp_file_name [](http://example.com/codeflow?start=64&length=12)

nit: `file_name`, similarly can remove the other "lp" and "h" prefixes as those are more of the "hungarian notation" used by Windows API's which have a short "type" prefix in variable names.